File: Python/1180.py
n=int(input())

a = input()

a = a.split()

for i in range(len(a)):
    a[i] = int(a[i])

menor = a[0]
posicao = 0
for j in range(1,len(a)):
    if a[j] < menor:
        menor = a[j]
        posicao = j

print('Menor valor: {}'.format(menor))
print('Posicao: {}'.format(posicao))

File: CSharp/1180.cs
using System;
using System.Linq;

class URI
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine()); 
        int[] x = Console.ReadLine().Trim().Replace("  "," ").Split().Select(int.Parse).ToArray(); 
        int min = x.Min();
        int indice = Array.IndexOf(x, min);

        Console.WriteLine($"Menor valor: {min}");
        Console.WriteLine($"Posicao: {indice}");
    }
}
